recipes/fts-rest/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "fts-rest" %}
+{% set version = "3.8.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.cern.ch/fts/fts-rest/-/archive/v{{ version }}/fts-rest-v{{ version }}.tar.gz
+  sha256: 91945cfc931f7d22f9586fd41939284e241a2e4911da1d2733a7328d46b131a3
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python <3
+    - pip
+  run:
+    - python <3
+    - pycurl
+    - m2crypto >=0.16
+    - requests
+    - sqlalchemy
+    - setuptools_scm
```

Comment:
Shouldn't this also be a `host` requirement?

Reply:
So far as I can tell it's only needed at runtime unless there is any "magic" that comes from having it in `host` that I'm unaware of?

Reply:
I've looked at https://github.com/cern-fts/fts-rest/blob/master/setup.py and cannot see that it requires `setuptools_scm` at all. Maybe this is a transitive dependency?

Reply:
It's needed for `pkg_resources` ([used here](https://github.com/cern-fts/fts-rest/blob/1dee4fa72d4b04d0a2c3cc0f56ff369d850dfd39/src/fts3/__init__.py)).

Reply:
`pkg_resources` is part of `setuptools`, not `setuptools_scm` ;)